### PR TITLE
14613 increase max URI and header length to Wildfly defaults

### LINKED ISSUE #14613: 414 Request-URI Too Long
### Describe the bug

When using Keycloak as IDP and Identity Server 4 with federated provider configuration, Keycloak returns `414 Request-URI Too Long` as the URL is more than 4k in length.

### Version

19

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by sschu**
Fixes #14613

### CODE REVIEW

**Review by vmuzikar — APPROVED** [review]
Thank you @sschu! LGTM.

Let's wait for @pedroigor approval as he was involved in discussions around this too.

**Review by pedroigor — APPROVED** [review]

**Pull request merged**
